src/main/java/net/mcreator/ui/dialogs/wysiwyg/InputSlotDialog.java

```diff
@@ -62,47 +63,45 @@ public InputSlotDialog(WYSIWYGEditor editor, @Nullable InputSlot slot) {
 			} catch (Exception exc) {
-				return new Validator.ValidationResult(Validator.ValidationResultType.ERROR, "Slot ID must be a number");
+				return new Validator.ValidationResult(Validator.ValidationResultType.ERROR, L10N.t("dialog.gui.slot_id_must_be_number"));
 			}
 			return new Validator.ValidationResult(Validator.ValidationResultType.PASSED, "");
 		});
 		slotID.setText("0");
-		options.add(PanelUtils.join(FlowLayout.LEFT, new JLabel("Slot ID: "), slotID));
+		options.add(PanelUtils.join(FlowLayout.LEFT, L10N.label("dialog.gui.slot_id"), slotID));
 
 		JColor color = new JColor(editor.mcreator);
 		options.add(PanelUtils.join(FlowLayout.LEFT,
-				new JLabel("<html>Custom color<br><small>Click only if you intend to use custom color: "), color));
+				L10N.label("dialog.gui.slot_custom_color"), color));
 
 		MCItemHolder limit = new MCItemHolder(editor.mcreator, ElementUtil::loadBlocksAndItems);
 		options.add(PanelUtils
-				.join(FlowLayout.LEFT, new JLabel("<html>Limit stack input to:<br><small>If empty, there is no limit "),
+				.join(FlowLayout.LEFT, L10N.label("dialog.gui.slot_limit_stack_input"),
 						limit));
 
-		JCheckBox disableStackInteraction = new JCheckBox(
-				"Disable player interaction with this slot (can't take item from it or insert in it)");
+		JCheckBox disableStackInteraction = L10N.checkbox("dialog.gui.slot_disable_player_interaction");
 		options.add(PanelUtils.join(FlowLayout.LEFT, disableStackInteraction));
 
-		JCheckBox dropItemsWhenNotBound = new JCheckBox(
-				"Drop items when GUI not bound to any external inventory is closed");
+		JCheckBox dropItemsWhenNotBound = L10N.checkbox("dialog.gui.slot_drop_item_when_gui_closed");
 		options.add(PanelUtils.join(FlowLayout.LEFT, dropItemsWhenNotBound));
 
 		dropItemsWhenNotBound.setSelected(true);
 
 		ProcedureSelector eh = new ProcedureSelector(IHelpContext.NONE.withEntry("gui/when_slot_changed"),
-				editor.mcreator, "When slot contents change", ProcedureSelector.Side.BOTH, false,
+				editor.mcreator, L10N.t("dialog.gui.slot_event_slot_content_changes"), ProcedureSelector.Side.BOTH, false,
 				Dependency.fromString("x:number/y:number/z:number/world:world/entity:entity/guistate:map"));
 		eh.refreshList();
 
 		ProcedureSelector eh2 = new ProcedureSelector(IHelpContext.NONE.withEntry("gui/when_slot_item_taken"),
-				editor.mcreator, "When item taken from slot", ProcedureSelector.Side.BOTH, false,
+				editor.mcreator, L10N.t("dialog.gui.slot_event_item_taken_from_slot"), ProcedureSelector.Side.BOTH, false,
 				Dependency.fromString("x:number/y:number/z:number/world:world/entity:entity/guistate:map"));
 		eh2.refreshList();
 
 		ProcedureSelector eh3 = new ProcedureSelector(IHelpContext.NONE.withEntry("gui/when_transferred_from_slot"),
-				editor.mcreator, "When transferred from slot (shift-click)", ProcedureSelector.Side.BOTH, false,
+				editor.mcreator, L10N.t("dialog.gui.slot_.event_transfered_from_slot"), ProcedureSelector.Side.BOTH, false,

```

Comment:
There is an extra dot in this key

Reply:
Thx, I'll look for eventual more mistakes before asking for new review

Reply:
Problem fixed as well!